SPECS/linux/linux-aws.spec

```diff
@@ -74,13 +74,8 @@ Patch52:        0006-btrfs-tree-checker-Enhance-chunk-checker-to-validate.patch
 Patch53:        0007-btrfs-tree-checker-Verify-inode-item.patch
 
 # Upgrade vmxnet3 driver to version 4
-Patch81:        0001-vmxnet3-prepare-for-version-4-changes.patch
-Patch82:        0002-vmxnet3-add-support-to-get-set-rx-flow-hash.patch
-Patch83:        0003-vmxnet3-add-geneve-and-vxlan-tunnel-offload-support.patch
-Patch84:        0004-vmxnet3-update-to-version-4.patch
-Patch85:        0005-vmxnet3-use-correct-hdr-reference-when-packet-is-enc.patch
-Patch86:        0006-vmxnet3-allow-rx-flow-hash-ops-only-when-rss-is-enab.patch
-Patch87:        0007-vmxnet3-use-correct-tcp-hdr-length-when-packet-is-en.patch
+Patch81:        0001-vmxnet3-version-4-driver.patch
```

Comment:
For every spec file you modify, you'll need to increment its release number and add a corresponding changelog, in order to generate new RPMs with your changes.

See these for example:
https://github.com/vmware/photon/commit/f308bbb854e946503e193ed7e69247d3459b0e1a
https://github.com/vmware/photon/commit/3ca4d2057ba99d703e96f0ee434a37d6db79c163